app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.kt

```diff
@@ -117,10 +117,6 @@ class FeedFragment : BaseListFragment<FeedState, Unit>() {
         }
     }
 
-    // /////////////////////////////////////////////////////////////////////////
```

Comment:
Bring back these comments.. sorry if I was unclear, I meant that old code shouldn't be commented-out but directly removed. But useful comments like this should be kept.